package.json

```diff
@@ -47,5 +51,26 @@
         "path": "./snippets/vba.json"
       }
     ]
+  },
+  "scripts": {
+    "vscode:prepublish": "npm run compile",
+    "compile": "tsc -p ./",
+    "lint": "eslint src --ext ts",
+    "watch": "tsc -watch -p ./",
+    "pretest": "npm run compile && npm run lint",
+    "test": "node ./out/test/runTest.js"
+  },
+  "devDependencies": {
```

Comment:
I presume these dev deps were from your own development, are they needed here?

I guess if there are other contributes it may help but I'm not sure of their purpose, could you clarify please?

Reply:
Again these are output from the Extension Generator.

> I guess if there are other contributes it may help...

Yes. I left them because I thought so.

- "pretest" is useful for other future developers
- "test": This is not needed now, because I did not implement test codes. But I left it to encourage for other future developers to implement test codes.
- "devDependencies": I am not sure they are need here.  I think I am using the latest tools and my code probably works with older tools.  I left this setting because I cannot test if my code works with older tools.

Feel free to remove some (or all) of them, if you don't this they are not needed.